share/spice/spotify/spotify.js

```diff
@@ -0,0 +1,39 @@
+(function (env) {
+    'use strict';
+    env.ddg_spice_spotify = function(api_result){
+
+        var query = DDG.get_query();
+        query = query.replace(/spotify/, ''); //replace trigger words from query
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.tracks.items.length === 0) {
+            return Spice.failed('spotify');
+        }
+
+        // Render the response
+        Spice.add({
+            id: 'spotify',
+
+            // Customize these properties
+            name: 'Spotify',
+            data: api_result.tracks.items,
+            meta: {
+                sourceName: 'Spotify',
+                sourceUrl: 'https://play.spotify.com/search/' + query.trim(),
+            },
+            normalize: function(item) {
+                return {
+                    title: item.name,
```

Comment:
@JMPerez apologies, I thought our template had a built-in subtitle but it doesn't. We can skip for now.

>Would I need to use a custom footer to display the album name like Soundcloud's?

Yes, if we want to add it, we'd need to use the Footer template like that.

> In that case, is there any CSS class to show ellipsis?

Theres a `one-line` class, but if you're dealing with text in Handlebars we have an `ellipses` helper. The `audio_item` template uses it for the title already: https://github.com/duckduckgo/duckduckgo-answerbar-templates/blob/master/item/audio_item.handlebars#L28